sitemedia/scss/components/_tabs.scss

```
@@ -2,35 +2,58 @@
 // Tabbed navigation that appears in the header of some pages.
 // -----------------------------------------------------------------------------
 
+@use "../base/colors";
 @use "../base/breakpoints";
 @use "../base/spacing";
 @use "../base/typography";
 
 .tabs {
-    display: flex;
-    justify-content: space-between;
-    text-align: center;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    align-content: stretch;
+    justify-content: center;
     margin: 0 auto;
     @include typography.nav-link;
     a,
     span {
-        flex: 1 0 auto;
-        display: inline-block;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        height: 100%;
+        padding-bottom: spacing.$spacing-xs;
+        border-bottom: 4px solid var(--tabs-bottom);
         @include breakpoints.for-tablet-landscape-up {
-            padding: spacing.$spacing-2xs spacing.$spacing-lg;
+            padding-left: spacing.$spacing-xl;
+            padding-right: spacing.$spacing-xl;
         }
     }
     a {
-        @include typography.link;
-    }
-    // Center tab padding boost
-    li:nth-child(2) a,
-    li:nth-child(2) span {
-        @include breakpoints.for-tablet-landscape-up {
-            padding: spacing.$spacing-2xs spacing.$spacing-2xl;
+        text-decoration: none;
+        &:focus-visible {
+            outline: 0.1rem solid var(--tabs-focus);
+            outline-offset: -0.33rem;
```

@blms I had an idea about using a pseudo-element to get the focus outline closer to the designs. I tried this out locally, see what you think.


```suggestion
      position: relative;
       /* use ::after for focus outline */
        &::after {
            content: "";
            width: 80%;
            height: 80%;
            position: absolute;
        }
        &:focus-visible {
            outline: none;
        }
        &:focus-visible::after {
            outline: 0.1rem solid var(--tabs-focus);
        }
```

---

This is great!! Excellent solution, Rebecca.